granulate_utils/metadata/databricks_client.py

```diff
@@ -34,55 +36,102 @@ def __init__(self, logger: logging.LoggerAdapter) -> None:
         else:
             self.logger.debug(f"Got Databricks job name: {self.job_name}")
 
+    def _request_get(self, url: str) -> requests.Response:
+        resp = requests.get(url, timeout=REQUEST_TIMEOUT)
+        resp.raise_for_status()
+        return resp
+
     @staticmethod
-    def get_webui_address() -> Optional[str]:
+    def get_webui_address() -> str:
         with open(DATABRICKS_METRICS_PROP_PATH) as f:
             properties = f.read()
-        host = dict([line.split("=", 1) for line in properties.splitlines()])[HOST_KEY_NAME]
+        try:
+            host = dict([line.split("=", 1) for line in properties.splitlines()])[HOST_KEY_NAME]
+        except Exception:
+            raise DatabricksJobNameDiscoverException(f"Failed to get Databricks webui address {properties=}")
```

Comment:
please note @michelhe @Jongy 
df6a584

Reply:
please note 2 
please re-review this https://github.com/Granulate/granulate-utils/commit/a78ddb3e520002b3fe9e370a06232d58c7511f10